build/lib/release.sh

```diff
@@ -391,38 +391,35 @@ function kube::release::package_salt_tarball() {
 function kube::release::package_kube_manifests_tarball() {
   kube::log::status "Building tarball: manifests"
 
-  local salt_dir="${KUBE_ROOT}/cluster/saltbase/salt"
+  local src_dir="${KUBE_ROOT}/cluster/gce/manifests"
 
   local release_stage="${RELEASE_STAGE}/manifests/kubernetes"
   rm -rf "${release_stage}"
 
-  mkdir -p "${release_stage}"
-  cp "${salt_dir}/kube-registry-proxy/kube-registry-proxy.yaml" "${release_stage}/"
```

Comment:
These files moved to a different dir in the tar, but I don't see a corresponding change in the node setup scripts.

Reply:
changes are in `cluster/gce/gci/configure-helper.sh`

Reply:
I see it now. thanks.